x11-wm/mutter/mutter-45_beta1.ebuild

```diff
@@ -0,0 +1,235 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
I don't know if there's something blocking add Py3.12, but we should remove 9 regardless.